### DISCUSSION TITLE
Pouch db crashes my React native App

### DISCUSSION THREAD

**Discussion by gbengeA** [pouchdb/pouchdb#8484] [General]
### Installation 
My App crashes on adding pouchDB code after installation 

### Info
- Environment: ( expo react-native  web and android)
- Platform: (Chrome, Android, iOS)
-  Target App to run on expo web, android and ios 
- 

### Reproduce 
 # dependecies 
"pouchdb-react-native": "^6.4.1",
 "expo": "~44.0.0",
  "react-native": "0.64.3"
# code 
```
import PouchDB from 'pouchdb-react-native'
 db.get('4711')
        .then(doc => console.log(doc))
```

# Crash logs
 ./node_modules/pouchdb-adapter-asyncstorage/src/databases.js:32
"export 'default' (imported as 'AsyncStorageCore') was not found in './asyncstorage_core'
  30 |   }
  31 |
> 32 |   const storage = new AsyncStorageCore(opts.name)
  33 |
  34 |   storage.multiGet(toMetaKeys([
  35 |     '_local_uuid', '_local_doc_count', '_local_last_update_seq'

**Comment by mqtik** [comment]
Do not use that package, it uses AsyncStorage, which's insane since it has no indexes. So, everytime you get, find, query, or whatever, it loads all documents into memory.

Use this one instead: https://github.com/craftzdog/pouchdb-react-native

**Reply by Productivix** [reply]
hello, 
yes but that last one is not working as well - see it issues
There is now at that time no solutions for React Native 0.70 .
